sdks/python/apache_beam/transforms/core.py

```diff
@@ -2315,7 +2331,8 @@ def from_runner_api(proto, context):
         windowfn=WindowFn.from_runner_api(proto.window_fn, context),
         triggerfn=TriggerFn.from_runner_api(proto.trigger, context),
         accumulation_mode=proto.accumulation_mode,
-        timestamp_combiner=proto.output_time)
+        timestamp_combiner=proto.output_time,
+        allowed_lateness=proto.allowed_lateness // 1000)
```

Comment:
This is lossy, and it's possible that this comes from other languages that support sub-second allowed lateness. 

`Duration(micros=proto.allowed_lateness * 1000)`.